src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("Hello, world!");
+}
```

Comment:
This does not look like it's up to date.

Reply:
It will be changed on the next pull request. (#11)